src/main/java/de/tum/cit/aet/job/domain/CustomField.java

```diff
@@ -0,0 +1,43 @@
+package de.tum.cit.aet.job.domain;
+
+import de.tum.cit.aet.application.domain.CustomFieldAnswer;
+import de.tum.cit.aet.core.domain.converter.StringListConverter;
+import de.tum.cit.aet.job.constants.CustomFieldType;
+import jakarta.persistence.*;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "custom_fields")
+public class CustomField {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "custom_field_id", nullable = false)
+    private UUID customFieldId;
+
+    @ManyToOne
+    @JoinColumn(name = "job_id")
+    private Job job;
+
+    private int sequence;
+
+    private CustomFieldType type;
```

Comment:
```suggestion
    @Column(name = "custom_field_type")
    private CustomFieldType type;
```
Suggested adding @Column(name = "custom_field_type") to match the database’s snake case naming convention

Reply:
Thanks, I will fix this